Ghidra/Processors/Xtensa/data/languages/xtensaInstructions.sinc

```diff
@@ -0,0 +1,1345 @@
+:nsa at, as is op2 = 0b0100 & op1 = 0 & ar = 0b1110 & as & at & op0 = 0 {
+    at = lzcount(~as);
+}
+
+# NSAU - Normalization Shift Amount Unsigned, pg. 462. (Count leading zeros)
+:nsau at, as is op2 = 0b0100 & op1 = 0 & ar = 0b1111 & as & at & op0 = 0 {
+    at = lzcount(as);
+}
+
+# OEQ.S - Compare Single Equal, pg. 463.
+:oeq.s br, fs, ft is op2 = 0b0010 & op1 = 0b1011 & br & fs & ft & op0 = 0 {
+    br = !nan(fs) && !nan(ft) && fs f== ft;
+}
+
+# OLE.S - Compare Single Ordered and Less Than or Equal, pg. 464
+:ole.s br, fs, ft is op2 = 0b0110 & op1 = 0b1011 & br & fs & ft & op0 = 0 {
+    br = !nan(fs) && !nan(ft) && fs f<= ft;
+}
+
+# OLT.S - Compare Single Ordered and Less Than, pg. 465.
+:olt.s br, fs, ft is op2 = 0b0100 & op1 = 0b1011 & br & fs & ft & op0 = 0 {
+    br = !nan(fs) && !nan(ft) && fs f< ft;
+}
+
+# MOV - Move, pg. 412. Special case of OR as, at, at.
+:mov ar, as is op2 = 0b0010 & op1 = 0 & ar & as & as = at & op0 = 0 {
+    ar = as;
+}
+
+# OR - Bitwise Logical Or, pg. 466.
+:or ar, as, at is op2 = 0b0010 & op1 = 0 & ar & as & at & op0 = 0 {
+    ar = as | at;
+}
+
+# ORB - Boolean Or, pg. 467.
+:orb br, bs, bt is op2 = 0b0010 & op1 = 0b0010 & br & bs & bt & op0 = 0 {
+    br = bs || bt;
+}
+
+# ORBC - Boolean Or with Complement, pg. 468.
+:orbc br, bs, bt is op2 = 0b0011 & op1 = 0b0010 & br & bs & bt & op0 = 0 {
+    br = bs || !bt;
+}
+
+# PDTLB - Probe Data TLB, pg. 469.
+:pdtlb at, as is op2 = 0b0101 & op1 = 0 & ar = 0b1101 & as & at & op0 = 0 {
+    at = pdtlb(as);
+}
+
+# PITLB - Probe Instruction TLB, pg. 470.
+:pitlb at, as is op2 = 0b0101 & op1 = 0 & ar = 0b0101 & as & at & op0 = 0 {
+    at = pitlb(as);
+}
+
+# QUOS - Quotient Signed, pg. 471.
+:quos ar, as, at is op2 = 0b1101 & op1 = 0b0010 & ar & as & at & op0 = 0 {
+    ar = as s/ at;
+}
+
+# QUOU - Quotient Unsigned, pg. 472.
+:quou ar, as, at is op2 = 0b1100 & op1 = 0b0010 & ar & as & at & op0 = 0 {
+    ar = as / at;
+}
+
+# RDTLB0 - Read Data TLB Virtual Entry, pg. 473.
+:rdtlb0 at, as is op2 = 0b0101 & op1 = 0 & ar = 0b1011 & as & at & op0 = 0 {
+    at = rdtlb0(as);
+}
+
+# RDTLB1 - Read Data TLB Entry Translation, pg. 474.
+:rdtlb1 at, as is op2 = 0b0101 & op1 = 0 & ar = 0b1111 & as & at & op0 = 0 {
+    at = rdtlb1(as);
+}
+
+# REMS - Remainder Signed, pg. 475.
+:rems ar, as, at, is op2 = 0b1111 & op1 = 0b0010 & ar & as & at & op0 = 0 {
+    ar = as s% at;
+}
+
+# REMU - Remainder Unsigned, pg. 476.
+:remu ar, as, at, is op2 = 0b1110 & op1 = 0b0010 & ar & as & at & op0 = 0 {
+    ar = as % at;
+}
+
+# RER - Read External Register, pg. 477.
+:rer as, at is op2 = 0b0100 & op1 = 0 & ar = 0b0110 & as & at & op0 = 0 {
+    as = rer(at);
+}
+
+# RET.N - Narrow Non-Windowed Return, pg. 479.
+:ret.n is n_ar = 0b1111 & n_as = 0 & n_at = 0 & n_op0 = 0b1101 {
+    return [a0];
+    o2=a2;
+}
+
+# RFDD - Return from Debug and Dispatch, pg. 484.
+:rfdd is op2 = 0b1111 & op1 = 0b0001 & ar = 0b1110 & (as = 0b0000 | as = 0b0001) & at = 0b0001 & op0 = 0 {
+    rfdd();
```

Comment:
I think all these `return from` might help disassembler and produce better decompilation with return something.  right now they'd just fall through, but they should have some sort of flow

might try something like this for all of these:
```diff
-    rfdd();
+    local tmp:4 = rfdd();
+    return [tmp];
```

Reply:
Addressed in c5e8f015e55a404bc30c7240e3ad8e2da8ea8c72. Thank you!